www/login.ts

```diff
@@ -0,0 +1,16 @@
+import {Request, Response} from 'express';
+import { getClient } from '../lib';
+import { getAccessToken } from '../lib/spotify/access-token';
+import { getMeInfo } from '../lib/spotify/me';
+
+export async function POST_login(req: Request, res: Response): Promise<void> {
+  const client = await getClient();
+
+  const refreshToken = req.body.refreshToken;
+  const accessToken = await getAccessToken(refreshToken);
+  const userInfo = await getMeInfo(accessToken);
+
+  const userTable = await client.openCollection('users');
```

Comment:
later i will create an object to clean up the user and make sure that we are indexing the DB with the user ID.. currently it is created by OCI...